easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.0.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -22,69 +22,105 @@ dependencies = [
     ('h5py', '2.9.0'),
 ]
 
-exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
 use_pip = True
 
 exts_list = [
-    ('protobuf-python', '3.9.2', {
-        'modulename': 'google.protobuf',
-        'patches': ['TensorFlow-1.13.1_protobuf-3.6.1.2_fix_26155.patch'],
-        'source_tmpl': 'protobuf-python-%(version)s.tar.gz',
-        'source_urls': ['https://github.com/google/protobuf/releases/download/v%(version)s/'],
-        'start_dir': 'python',
-        'use_pip': False,
-        'checksums': [
-            'a7564e52d2842033303d2a81b4b8099145b5aabe0a5531a8b057801377bc2a13',  # protobuf-python-3.9.2.tar.gz
-            # TensorFlow-1.13.1_protobuf-3.6.1.2_fix_26155.patch
-            'db514d3698d37e329cecdd2245dcd6ff65f2b739a66730f303e8a9f6c107392b',
-        ],
+    # tensorboard 2.0.0 has requirement setuptools>=41.0.0
```

Comment:
@Flamefire Please include these additional comments here:

```
# must stick to setuptools < 41.4 to avoid problems with astor,
# see https://github.com/berkerpeksag/astor/issues/162
```

Reply:
👍 Done